catalog/view/theme/default/template/payment/razorpay.tpl

```diff
@@ -32,6 +32,11 @@
       }
     } else {
       if(!razorpay_instance){
+        <?php if ($display_currency !== $currency_code) { ?>
```

Comment:
shift this entire check to javascript. doesn't look nice to have a php conditional inside a JS conditional